.test/great_expectations/expectations/mobile/v1/mobile_sessions_screen_view_details.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
I can't find where this is used?

Reply:
Good spot. This was used during dev but not anymore. Will remove